### PR TITLE
Small fixes to the options page

### PR DESCRIPTION
**Pull request by stefansundin**
Hello, I'm back with some small fixes that I have accumulated over a few months.

It was possible to get the "empty message row" to show up when having things in the table, for example in the list of connected databases. If you add your first connection from the options page then you'll get this:

Screenshot 2023-12-22 at 11 06 43

So I rewrote the logic that hides the row in pure CSS, which makes the code much cleaner and fixes this bug.

---

I removed `table-bordered` from the tables since I think it gives the pages a cleaner look. Here's the before/after:

Screenshot 2023-12-22 at 12 06 52

Screenshot 2023-12-22 at 12 06 59

Hopefully you like this look better too. 😄

---

Also fixed some other things, please see the commit messages for more details. Let me know if you want me to change anything.

### CODE REVIEW

**Comment by varjolintu** [comment]
Thank you! This is an old bug I never had time to fix :)

File: keepassxc-browser/options/options.html [hunk 1]
```diff
@@ -543,7 +543,7 @@ <h2 class="pb-3 mt-0" data-i18n="optionsConnectedDatabasesTab"></h2>
               <div class="card my-4 shadow">
                 <div class="card-body">
                   <div class="table-responsive">
-                    <table class="table table-sm table-striped table-bordered">
+                    <table class="table table-sm table-striped">
```
Comment (varjolintu) [inline keepassxc-browser/options/options.html:546]: Why is `table-bordered` removed from every table? After the change there are no borders in left or right.

Comment (stefansundin) [reply keepassxc-browser/options/options.html:546]: I think it gives the pages a cleaner look, but if you feel strongly about it then I can revert this change.

Comment (varjolintu) [reply keepassxc-browser/options/options.html]: I understand your point. Maybe bigger design changes should be done in a separate PR anyway.

**Review by varjolintu — APPROVED** [review]

**Review by droidmonkey — APPROVED** [review]

**Comment by droidmonkey** [comment]
Personally don't like tables without side borders, they tend to just melt into the background which isn't the intent of a table. I think that table (or all tables) could use a better, darker, border color in dark mode instead.

**Pull request merged**